### PR TITLE
Fix the issue of update the wrapMode after changing the image type

### PR DESCRIPTION
**Pull request by naturalch**
Re: #
https://github.com/cocos/3d-tasks/issues/15635

### Changelog

*

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [x] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

File: editor/inspector/assets/image.js [hunk 1]
```diff
@@ -327,26 +327,22 @@ exports.methods = {
...
+            // use default wrapMode if not changed
+            const wrapModeName = this.meta.userData.type === 'texture' ? 'Repeat' : 'Clamp';
+            this.metaList.forEach(async (meta) => {
+                const data = ModeMap.wrap[wrapModeName];
+                if (!meta.subMetas[textureKey]) {
+                    meta.subMetas[textureKey] = {
+                        userData: {}
                     }
+                }
+                let wrapModeCache = await Editor.Profile.getConfig('inspector', `${meta.uuid}@${textureKey}.texture.wrapMode`, 'default');
```
Comment (naturalch) [inline editor/inspector/assets/image.js:339]: For multiple selections, it is necessary to check all picture, In the case of selecting multiple images where two images have caching and one does not.

**Review by yanOO1497 — APPROVED** [review]

File: editor/inspector/assets/texture/texture.js [hunk 2]
```diff
@@ -413,7 +413,9 @@ const Elements = {
                 // 根据 wrapModeSelect 组合值同步相应的 wrapModeS/wrapModeT 到 userData
                 const value = event.target.value;
                 // 临时记录用户的修改配置
-                Editor.Profile.setConfig('inspector', `${this.meta.uuid}.texture.wrapMode`, value, 'default');
+                this.metaList && this.metaList.forEach((meta) => {
+                    Editor.Profile.setConfig('inspector', `${meta.uuid}.texture.wrapMode`, value, 'default');
```
Comment (VisualSJ) [inline editor/inspector/assets/texture/texture.js:417]: Is this data for the interface temporary? Would it be better to put it in 'Temp'?
这个数据是界面用的临时数据吧？放到 Temp 会不会更好？

File: editor/inspector/assets/image.js [hunk 3]
```diff
@@ -327,26 +327,22 @@ exports.methods = {
...
-                        meta.subMetas[textureKey] = {
-                            userData: {}
-                        }
+            // use default wrapMode if not changed
+            const wrapModeName = this.meta.userData.type === 'texture' ? 'Repeat' : 'Clamp';
+            this.metaList.forEach(async (meta) => {
+                const data = ModeMap.wrap[wrapModeName];
+                if (!meta.subMetas[textureKey]) {
+                    meta.subMetas[textureKey] = {
+                        userData: {}
```
Comment (VisualSJ) [inline editor/inspector/assets/image.js]: This needs to end with a comma here.

File: editor/inspector/assets/texture/texture.js (hunk 2 above)
Comment (naturalch) [reply editor/inspector/assets/texture/texture.js:417]: Temporary data is stored in the 'default' and is only valid for the current session. It won't be remembered next time. So, there is no need to store it in 'temp' for long-term recording.

**Review by VisualSJ — APPROVED** [review]

**Pull request merged**